packages/nextjs-kit/src/components/recipe.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Image from 'next/image';
+import type { ImageProps } from 'next/image';
+import { useRouter } from 'next/router';
+
+interface RecipeProps {
+	title: string;
+	category: string;
+	imageProps?: ImageProps;
+	ingredients: string[];
+	instructions: string;
+	children?: JSX.Element;
+}
+
+/**
+ *
+ * @param props - The props needed for the Recipe component
+ * @param props.title - The title of the recipe
+ * @param props.category - The identifying category of your recipe
+ * @param props.imageProps - All props the user wishes to pass to the next/image component @see {@link https://nextjs.org/docs/api-reference/next/image} for all next/image documentation
+ * @param props.ingredients - An array of strings with ingredients to be displayed
+ * @param props.instructions - Instructions from a CMS, usually a string of raw HTML. This string will be set as `dangerouslySetInnerHTML`
+ * @returns A recipe component with content and an optional image passed by the user
+ * @remarks
+ * imageProps is an optional prop to be used if there is an image to be associated with the content.
+ * If imageProps is used it is required that the user passes in value for the images src
+ */
+const Recipe: React.FC<RecipeProps> = ({
+	title,
+	category,
+	imageProps,
+	ingredients,
+	instructions,
+	children,
+}: RecipeProps) => {
+	const router = useRouter();
+	return (
+		<article className="prose lg:prose-xl mt-10 mx-auto h-fit p-4 sm:p-0">
+			<header>
+				<h1>{title}</h1>
+				<div className="flex flex-row justify-between">
+					<a
+						onClick={() => router.back()}
+						className="font-normal cursor-pointer"
+					>
+						Back &rarr;
+					</a>
+					<span className="text pb-2 pr-3 text-sm text-slate-400">
+						{category}
+					</span>
+				</div>
+			</header>
+			{imageProps ? (
+				<div className="relative max-w-lg mx-auto min-w-full h-[50vh] rounded-lg shadow-lg overflow-hidden mt-12 mb-10">
+					<Image {...imageProps} />
+				</div>
+			) : null}
```

Comment:
Check the refactor on the ContentWithImage component. I think if we are going to keep it as a cover image type thing, we should be consistent with the props we take in.

I think it could even make sense to extract the `featuredImage` component into an atom that can be used in this Recipe component, as well as in ContentWithImage and any future components with a 'featuredImage', but perhaps we should wait on that.